connexion/apps/aiohttp_app.py

```diff
@@ -90,6 +90,6 @@ def run(self, port=None, server=None, debug=None, host=None, **options):
             if options.get('use_default_access_log'):
```

Comment:
I'm not sure what the rationale was to manage these explicitly, but it seems like we should do one, or the other, not both.

Reply:
git history shows that this was part of the very first commit that added aiohttp: 5ce51450fe21927673727d1284a110342f499075

So, nothing enlightening to add there. It is a little odd...